src/components/Calculator.jsx

```diff
@@ -0,0 +1,52 @@
+const Calculator = () => {
```

Comment:
- [ ] As pointed out in the requirements, the Calculator component should have at least one child component hence kindly refactor your code once more :+1: